includes/discovery/toner.inc.php

```diff
@@ -58,6 +58,38 @@
             );
         }
     }
+
+    echo PHP_EOL, 'Tray Paper Level', PHP_EOL;
+    $tray_oids = snmpwalk_cache_oid($device, 'prtInputName', [], 'Printer-MIB');
+    if (!empty($tray_oids)) {
+        $tray_oids = snmpwalk_cache_oid($device, 'prtInputCurrentLevel', $tray_oids, 'Printer-MIB');
+        $tray_oids = snmpwalk_cache_oid($device, 'prtInputMaxCapacity', $tray_oids, 'Printer-MIB');
+    }
+
+    d_echo($tray_oids);
+    foreach ($tray_oids as $index => $data) {
+        $last_index = substr($index, strrpos($index, '.') + 1);
+        $type = 'jetpapertray';
```

Comment:
Can probably drop jet here and just go papertray (jetdirect above is for legacy reasons)